src/code/z_quake.c

```diff
@@ -43,7 +43,7 @@ f32 Quake_Random(void) {
     return 2.0f * (Rand_ZeroOne() - 0.5f);
 }
 
-void Quake_UpdateShakeInfo(QuakeRequest* req, ShakeInfo* shake, f32 y, f32 x) {
+void Quake_UpdateShakeInfo(QuakeRequest* req, ShakeInfo* shake, f32 yOffset, f32 xOffset) {
```

Comment:
The problem with `offset` is we usually use that to mean the difference between 2 points, from what you said earlier I had the impression these were magnitudes?

Reply:
The reference point here would be the screen coordinates without any quake i.e. the camera position I believe. And then the offset position would be the second point?